test/Option.test.ts

```diff
@@ -1,8 +1,11 @@
 import { describe, it, expect, vi } from "vitest"
-import { Option } from "../src/Option"
+import * as Option from "../src/Option"
 import { pipe } from "../src/composition"
 import { EqualityComparer } from "../src/EqualityComparer"
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
I think it's actually disabled in the entire `Nullable.ts` file, isn't it? 🤔 On this note though, I was thinking it may be helpful to have an alias in `prelude` like `export type NonNullish = {}` so that we don't have to disable the ban-types rule in so many spots. Thoughts?

Reply:
Went ahead with the `NonNullish` change